tpgtools/overrides/compute/samples/networkfirewallpolicyrule/global.tf.tmpl

```diff
@@ -0,0 +1,41 @@
+resource "google_compute_network_firewall_policy" "basic_network_firewall_policy" {
+  name = "{{policy}}"
+  project = "{{project}}"
+  description = "Sample global network firewall policy"
+}
+
+resource "google_compute_network_firewall_policy_rule" "primary" {
+ firewall_policy = google_compute_network_firewall_policy.basic_network_firewall_policy.name
+ action = "allow"
+ direction = "INGRESS"
+ priority = 1000
+match {
+ src_secure_tags {
+ name = "tagValues/${google_tags_tag_value.basic_value.name}"
```

Comment:
`match.src_secure_tags.name` is `Optional` but seems like it should be `Required`.  Can you confirm that it would make sense to specify `match { src_secure_tags {} }`? Even better, setting that in a test case would be great if it is a valid configuration, to make sure that it works correctly! Empty blocks can work unusually.

Reply:
fixed in latest DCL 1.26.4